src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,58 @@
+package edu.kis.vh.nursery;
+
+//TODO: think about extracting interface and using bridge.
+public class DefaultCountingOutRhymer {
+
+	private static final int EMPTY = -1;
+	private static final int NUMBERS_SIZE = 12;
+	private static final int FULL = 11;
```

Comment:
This constant is related to NUMBERS_SIZE. Changing only one leads to an error (inconsistency).